app/src/test/java/org/fossasia/openevent/app/unit/model/FaqRepositoryTest.java

```diff
@@ -0,0 +1,8 @@
+package org.fossasia.openevent.app.unit.model;
+
+/**
+ * Created by priyanshu on 2/3/18.
+ */
+
+public class FaqRepositoryTest {
+}
```

Comment:
This file is empty

Reply:
Sorry for that. I've updated the PR, Please have a look

Reply:
Sorry for that. I've updated the PR, please have a look.